ambuda/models/auth.py

```diff
@@ -0,0 +1,90 @@
+from flask_login import UserMixin
+from enum import Enum
+from datetime import datetime
+from sqlalchemy import (
+    Column,
+    DateTime,
+    Integer,
+    String,
+)
+from sqlalchemy.orm import relationship
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from ambuda.models.base import Base, foreign_key, pk
+
+
+class SiteRole(Enum):
+    PROOFREADER = "Allow proofreading of texts prior."
+    SITE_ADMIN = "Allow management of site data."
+
+
+class UserRoles(Base):
+
+    """Roles for each user"""
+
+    __tablename__ = "user_roles"
+
+    #: Primary key.
+    id = pk()  # TODO: make pk both columns
```

Comment:
I tihnk you should be able to do this by adding `primary_key=True` to both IDs and deleting `id` ?